gcloud/storage/test_blob.py

```diff
@@ -488,11 +518,14 @@ def test_upload_from_file_w_slash_in_name(self):
         blob = self._makeOne(BLOB_NAME, bucket=bucket)
         blob._CHUNK_SIZE_MULTIPLE = 1
         blob.chunk_size = 5
-        with NamedTemporaryFile() as fh:
-            fh.write(DATA)
-            fh.flush()
-            blob.upload_from_file(fh, rewind=True)
-            self.assertEqual(fh.tell(), len(DATA))
+
+        with _NamedTemporaryFile() as temp:
+            with open(temp.name, 'wb') as file_obj:
```

Comment:
I'm puzzled here:  `tempfile.mkstemp` actually opens the file and returns a handle to it (which the class discards by assigning to `_`:  I would expect Windows to barf about reopening the unclosed file.